configs/server/forseti_conf_server.yaml.in

```diff
@@ -212,6 +212,8 @@ scanner:
           enabled: true
         - name: cloudsql_acl
           enabled: true
+        - name: config_validator
+          enabled: false
```

Comment:
This is a early release version, maybe we can default to enable in a newer version? the config validator binary has some limitation to it at this moment (e.g. if violations > 4 Mbs the communication would fail)

Reply:
Okay. We should document the limitations.